packages/web-frontend/src/components/Marker/markerColors.js

```diff
@@ -107,24 +123,26 @@ export function getTimeHeatmapColor(value, noDataColour) {
  * https://commons.wikimedia.org/wiki/File:South_Africa_2011_population_density_map.svg
  *
  * @param {number} value A value in the range [0..1] representing a percentage
+ * @param {default} if default is true, return lightest to darkest colour, otherwise return darkest to lightest
  * @returns {style} css rgb string, e.g. `rgb(0,0,0)`
  */
-export function getHeatmapColor(value) {
-  let rgb = [0, 0, 0];
-
-  if (value < 0.15) rgb = [255, 255, 204];
-  else if (value >= 0.15 && value < 0.25) rgb = [255, 237, 160];
-  else if (value >= 0.25 && value < 0.35) rgb = [254, 217, 118];
-  else if (value >= 0.35 && value < 0.45) rgb = [254, 178, 76];
-  else if (value >= 0.45 && value < 0.55) rgb = [253, 141, 60];
-  else if (value >= 0.55 && value < 0.65) rgb = [252, 78, 42];
-  else if (value >= 0.65 && value < 0.75) rgb = [227, 26, 28];
-  else if (value >= 0.75 && value < 0.85) rgb = [118, 0, 38];
-  else if (value >= 0.85) rgb = [128, 0, 38];
+function getHeatmapColorByOrder(value, swapColor) {
+  const difference = value - 0.15;
+  const index = difference < 0 ? 0 : Math.floor(difference / 0.1) + 1;
+  const indexInRange = index > rgbSet.length - 1 ? rgbSet.length - 1 : index;
 
+  const rgb = !swapColor ? rgbSet[indexInRange] : rgbSet[rgbSet.length - indexInRange - 1];
```

Comment:
Nice one! And would aid if later we wanted a configurable scale.

Reply:
> Looks good Biao!
> Just as an optional naming suggestion, I would replace `swap` with `reverse` or `reversed` just because it feels closer to me. `REVERSE_DEFAULT_COLOR_SCHEME` ?
> Feel free to push back on this :)

Thanks Nick! Will do.